scheme/common/cca/realm/realm_utils.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cca
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/veraison/ccatoken"
+	"github.com/veraison/services/log"
+)
+
+func GetRim(subscheme string, attr json.RawMessage) (string, error) {
```

Comment:
As with field names, these should probably be `GetRIM`, `GetRPV`, etc...

Reply:
Now have capitalized RIM, REM and RPV in the code base.